documentation/commands.md

```diff
@@ -254,3 +267,47 @@ be used to "shortstem" the aircraft, which puts the data block right on top of
 the aircraft's position symbol.
 
 _Syntax -_ ``AAL123 `2``
+
+### Version
+
+_Information -_ Displays the version of the game running.
+
+Example: `Air Traffic Control simulator version 5.3.0`
+
+_Syntax -_ `version`
+
+### Pause
+
+_Information -_ Pauses the game. Click anywhere to resume.
+
+_Syntax -_ `pause`
+
+### Tutorial
+
+_Information -_ Opens the tutorial.
+
+_Syntax -_ `tutorial`
+
+### Timewarp
+
+_Information -_ Sets the rate at which time passes, normal is `1`. While the time warp button can only set the rate to `1`, `2`, or `5`, the command accepts any number.
+
+_Parameters -_ A number to multiply the rate at which time passes. `1` resets to normal time.
+
+_Syntax -_ `timewarp [rate]`
+
+### Rate
```

Comment:
I didn't think so either, but I thought I just wasn't following the code. Thanks